packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -77,6 +76,44 @@ const ALLOWED_INPUT_TYPES = ['text', 'number', 'password', 'email', 'tel', 'url'
 
 const ALLOW_VIRTUAL_ELEMENTS = false;
 
+/**
+ * Raises a click on a target element based on a keyboard event.
+ */
+function _raiseClickFromKeyboardEvent(target: Element, ev?: React.KeyboardEvent<HTMLElement>): void {
+  let event;
+  if (typeof MouseEvent === 'function') {
```

Comment:
We don't support IE11 in N*, so we are safe to use constructors (I mean `new MouseEvent`). So it makes sense to this condition and use `new MouseEvent()`.